api-client/src/endpoints/createEventsEndpoint.ts

```diff
@@ -101,8 +105,8 @@ const createRecurringEvents = (event: JsonEventType): JsonEventType[] => {
     return [event]
   }
   const rrule: RRule = rrulestr(event.recurrence_rule)
-  const today = moment().toDate()
-  const lastValidDay = moment().add(MAX_FUTURE_EVENT_IN_MONTHS, 'months').toDate()
+  const today = DateTime.now().toJSDate()
+  const lastValidDay = DateTime.now().plus({ months: MAX_FUTURE_EVENT_IN_MONTHS }).toJSDate()
```

Comment:
@steffenkleinle @sarahsporck I think I need to get rid of the **rrule** library, because it is based on Date

Reply:
If that library requires Date, you can just use normal JS dates there. No need to remove the library or similar but also we shouldn't use normal JS date anywhere unless necessary.

Reply:
@steffenkleinle in luxon documentation it is mentioned for moment users that `toDate()` should be replaced with `toJSDate()` [](https://moment.github.io/luxon/#/moment)
And I can't replace it because of the error `TS2551: Property 'toDate' does not exist on type 'DateTime'. Did you mean 'toJSDate'?`

Reply:
I think this is fixed, right? If the doc says you should use `toJSDate()` and you are using that now that should be fine right?